src/templates/settings.html

```diff
@@ -317,6 +317,18 @@ <h4>Your Info:</h4>
     </div>
     <br />
     <br />
+    <div class="select center">
```

Comment:
I would add 2 break lines above here to space out the dropdown from the above button.

Reply:
![image](https://user-images.githubusercontent.com/11899221/54978354-1e2e9900-4fc6-11e9-8d97-db9ebfe54d2f.png)

Reply:
:+1: